multiqc/multiqc.py

```diff
@@ -176,6 +176,14 @@
 @click.option("-q", "--quiet", is_flag=True, help="Only show log warnings")
 @click.option("--profile-runtime", is_flag=True, help="Add analysis of how long MultiQC takes to run to the report")
 @click.option("--no-ansi", is_flag=True, help="Disable coloured log output")
+@click.option(
+    "--custom-css-files",
+    "--custom-css-files",
+    "custom_css_files",
+    type=click.Path(exists=True, readable=True),
+    multiple=True,
+    help="Custom CSS files to add to the final report",
```

Comment:
One per use of the flag, so I prefer singular here. 

```suggestion
    help="Custom CSS file to add to the final report",
```